events-api/src/modules/events/events.service.ts

```diff
@@ -1,44 +1,27 @@
 import { BadRequestException, Injectable, NotFoundException } from '@nestjs/common';
-import { DatabaseService } from '../database/database.service';
 import { type CreateEventReqDto } from './dto/requests';
-import { time } from 'src/shared/libs/time.lib';
-import { ListEventsParamsReqDto } from './dto/requests/list-events-params.dto';
+import { type ListEventsParamsReqDto } from './dto/requests/list-events-params.dto';
+import { EventsRepository } from './events.repository';
+import { UpdateEventReqDto } from './dto/requests/update-event.dto';
 
 @Injectable()
 export class EventsService {
-  constructor(private readonly databaseService: DatabaseService) {}
+  constructor(private readonly eventsRepository: EventsRepository) {}
```

Comment:
Just a sidenote, with Prisma repository pattern is not that popular since whole Prisma client is almost like a big repository. But personally I am good with way how you did it.